frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-request-data.ts

```diff
@@ -48,6 +49,12 @@ export const getOCSRequestData = (
   if (provisioner === NO_PROVISIONER) {
     requestData.spec.monDataDirHostPath = '/var/lib/rook';
     requestData.spec.storageDeviceSets[0].portable = false;
+    requestData.metadata = {
+      ...requestData.metadata,
+      annotations: {
+        [PLATFORM_ANNOTATION]: 'true',
+      },
+    };
```

Comment:
nit: We should these as args of the request data generator function. No point in having a function if we have to modify again and again.

Reply:
I could not do refactors for this PR. Just using an existing functionality.

Reply:
Not relevant to this PR, but I think we should make the request function generic.
Post the install PR or in the install PR itself , it can be considered @gnehapk ^^